cli/cmd/chunkIntervalGet.go

```diff
@@ -27,7 +29,20 @@ func chunkIntervalGet(cmd *cobra.Command, args []string) error {
 
 	metric := args[0]
 
-	pool, err := pgconn.OpenConnectionToDB(namespace, name, user, dbname, FORWARD_PORT_TSDB)
+	secretKey, user, err := utils.GetDBSecretKeyAndDBUser(helmReleaseName, user)
+	if err != nil {
+		return fmt.Errorf("could not get DB secret key from gelm release: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("could not get DB secret key from helm release: %w", err)
```

saw gelm a few times, I will only leave one comment